src/libraries/Microsoft.Extensions.Primitives/src/InplaceStringBuilder.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Primitives
+{
+    [DebuggerDisplay("Value = {_value}")]
```

Comment:
>[DebuggerDisplay("Value = {_value}")] [](http://example.com/codeflow?start=4&length=37)

it is interesting to keep the debug display while we are obsoleting the struct. I am fine either way though.

Reply:
I just pulled this file exactly as it was from the last shipping release.  https://github.com/dotnet/extensions/blob/release/3.1/src/Primitives/src/InplaceStringBuilder.cs